lib/support/phone.rb

```diff
@@ -3,6 +3,12 @@ class Phone
 
     attr_accessor :country, :number
 
+    CODE_LENGTH_RANGE_BY_COUNTRY_CODE = {
```

Comment:
let's try to avoid global vars if possible? this definitely seems like a constant to me